web-console/src/lib/functions/ddl.ts

```diff
@@ -216,7 +216,7 @@ export const xgressJSONToSQLRecord = (
 ): Record<string, SQLValueJS> =>
   Object.fromEntries(
     relation.fields.map(field =>
-      tuple(getCaseIndependentName(field), xgressJSONToSQLValue(field.columntype, value[field.name]))
+      tuple(getCaseIndependentName(field), xgressJSONToSQLValue(field.columntype, value[getCaseIndependentName(field)]))
```

Comment:
It looks to me like you want to add at least an unit test for this function which uses the relation in the bug report